packages/patternfly-4/gatsby-theme-patternfly-org/global.css

```diff
@@ -9,7 +9,7 @@
 }
 /* Please leave this file alone. */
 
-/* Qualtrics survey */
+/* Qualtrics surveys */
 .QSIPopOver iframe {
```

Comment:
Can we put this in `packages/patternfly-4/qualtrics.css` and then `import './qualtrics.css';` in `gatsby-browser.js`?